ui/src/tempVars/utils/replace.ts

```diff
@@ -0,0 +1,114 @@
+import {Template, TemplateValueType, TemplateValue} from 'src/types/tempVars'
+import {TEMP_VAR_INTERVAL} from 'src/shared/constants'
+
+export const replaceInterval = (
+  query: string,
+  pixels: number,
+  durationMs: number
+) => {
+  if (!query.includes(TEMP_VAR_INTERVAL)) {
+    return query
+  }
+
+  if (!pixels) {
+    pixels = 333
```

Comment:
yea definitely feel we should extract these constants

Reply:
consider it done.  we used to have a linter rule for magic numbers and it was removed.  mayhaps we add it back?